images/win/scripts/SoftwareReport/SoftwareReport.Helpers.psm1

```diff
@@ -102,4 +102,24 @@ function New-MDNewLine {
     )
     $newLineSymbol = [System.Environment]::NewLine
     return $newLineSymbol * $Count
-}
\ No newline at end of file
+}
+
+function Get-Link-Target {
+    param (
+        [string] $inputPath
+    )
+    $link = Get-Item $inputPath| Select-Object -ExpandProperty Target
+    If ($link) {
+      return " -> $link"
+    } Else {
+      return ""
+    }
+}
+
+function Get-Path-With-Link {
```

Comment:
Could you please rename the function to follow PowerShell naming?
`Get-PathWithLink`